### DISCUSSION TITLE
Questions about Github School Education credentials

### DISCUSSION THREAD

**Discussion by Noel5027** [community/community#143879] [GitHub Education]
### Hi everyone,

Earlier, I am Korean, so my English is bad. I am using a translator, so the sentences may be a bit awkward.

I'm trying to register a photo when registering my github eucation credentials, but the file upload button doesn't appear, and when I switch to the rear camera, only a white screen appears. I'm only allowed to use the front camera. What should I do in this case? All iPads, iPhones and laptops are the same.
![IMG_1853](https://github.com/user-attachments/assets/27ddc510-eda7-4bc4-8b9b-da868c808422)
![IMG_1854](https://github.com/user-attachments/assets/c993240c-a0a4-48c0-816b-5b4fb8b69d12)

**Comment by mastertanzeel** [comment]


**Reply by Noel5027** [reply]
I've already tried everything you said

**Comment by lllHunterlllSV** [comment]
**Hello! 👋**

Thank you for sharing your issue! It sounds like you're encountering some technical difficulties while trying to register your GitHub Education credentials. Let’s go through a few steps that could help resolve the problem:

- 📸 **Camera Access**: It seems like the camera settings might be restricting access. Make sure that the browser or app you're using has permission to access the camera. You can usually check this in the settings of your device:
    - **iPhone/iPad**: Go to *Settings* > *Privacy* > *Camera*, and ensure the browser or app you're using has camera access enabled.
    - **Laptop**: Make sure your browser has the correct permissions for camera use. You can check this by clicking the camera icon in the browser's address bar when you're on the GitHub Education page.

- 🖼️ **File Upload Button Not Appearing**: Sometimes, the **file upload button** may not show up due to browser issues. Try these solutions:
    - Ensure you’re using an up-to-date browser (like the latest version of **Safari**, **Chrome**, or **Firefox**).
    - Clear your browser cache or try using **Incognito/Private Mode**.
    - Try switching to a different browser or device to see if the issue persists.

- 🔄 **Switching to Front Camera**: If you’re only able to use the **front camera**, there could be a restriction in the app or browser you're using. Since the rear camera isn’t working and only a white screen appears, it might be a compatibility issue with the web interface. Unfortunately, GitHub Education might have limitations on which camera is supported during the registration process. You could try using a different browser or device to check if it’s a device-specific issue.

- 💬 **Contact GitHub Support**: If none of these steps resolve the issue, I recommend reaching out to [GitHub Support](https://support.github.com/). They can provide specific troubleshooting steps and might be able to offer a workaround for your situation.

I hope this helps, and good luck with your GitHub Education registration! 😊
